### PR TITLE
[feature](statistics)Support analyze rollup.

### PR DESCRIPTION
**Pull request by Jibing-Li**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java [hunk 1]
```diff
@@ -501,7 +501,8 @@ public List<Long> getMvColumnIndexIds(String columnName) {
         for (MaterializedIndexMeta meta : getVisibleIndexIdToMeta().values()) {
             Column target = meta.getColumnByDefineName(columnName);
             if (target != null) {
-                ids.add(meta.getIndexId());
+                // Return -1 for base index. This is for compatible with older version of column stats.
+                ids.add(meta.getIndexId() == baseIndexId ? -1 : meta.getIndexId());
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:505]: It seems that this interface is specifically designed for statistical information. Please add some comments to clarify its purpose and usage.

Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:510]: added

**Comment by Jibing-Li** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**